src/xo-app/new-vm/index.js

```diff
@@ -1250,14 +1260,10 @@ export default class NewVm extends BaseComponent {
           </Item>
         </SectionContent>,
         <SectionContent>
-          <Item>
-            {_('newVmMultipleVms')}
-            &nbsp;&nbsp;
+          <Item label='newVmMultipleVms'>
```

Comment:
I don't like the fact that `Item` is handling the translation, it should receive a React component.